next/.env.example

```diff
@@ -1,8 +1,16 @@
+# When developing locally, you should use node indside lando, which
```

Comment:
this should make it clearer that you should use lando instead of .env files + localhost

Reply:
Good idea. Another option would be to remove this file entirely - is there a reason we want to keep it?

Reply:
good point, removed in 4b2f0fdb5aef796b232ed9d9661b8c59309c0af7